polaris-react/src/components/SortIcon/SortIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import {CaretDownMinor, CaretUpMinor} from '@shopify/polaris-icons';
+import React from 'react';
+
+import {classNames} from '../../utilities/css';
+import type {SortDirection} from '../DataTable';
+import {Icon} from '../Icon';
+import {VisuallyHidden} from '../VisuallyHidden';
+
+import styles from './SortIcon.scss';
+
+interface SortIconProps {
+  sortDirection: SortDirection;
+  accessibilityLabel: string;
+}
+
+export function SortIcon({sortDirection, accessibilityLabel}: SortIconProps) {
+  return (
+    <div className={styles.SortIcon}>
```

Comment:
I noticed the current icon is implemented using `span` tags and it seems pretty common in other libraries, as well. One benefit of keeping HTML elements the same is tat they behave similarily when placed next to each other. Just based on the name `SortIcon` I would expect the markup to be similar to our other icons which all render out as `span`s.

<img width="858" alt="Screen Shot 2022-05-31 at 10 07 31 AM" src="https://user-images.githubusercontent.com/4960217/171193267-853f8294-52c4-4f39-aa02-e67051c4d054.png">